transport/http/config_test.go

```diff
@@ -530,8 +532,10 @@ func TestFallbackHandler(t *testing.T) {
 
 	dis := yarpc.NewDispatcher(cfg)
 
-	dis.Start()
-	defer dis.Stop()
+	require.NoError(t, dis.Start())
+	defer func() {
+		require.NoError(t, dis.Stop())
```

Comment:
defer should be assert because it might be called in case of `t.Fatal`, at which point requesting goroutine-exit again will skip the other defers.